src/pages/MyReservations/MyReservations.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import CarCard from '../../components/Cars/CarCard';
+import getCars from '../../redux/actions/Car/getCars';
+import getReservations from '../../redux/actions/Reservation/getReservation';
+
+function MyReservations() {
+  const dispatch = useDispatch();
+  const { reservation } = useSelector((state) => state.reservation);
+  const { cars } = useSelector((state) => state.cars);
+
+  useEffect(() => {
+    dispatch(getReservations());
+    dispatch(getCars());
+  }, [dispatch]);
+
+  const car = (id) => cars.filter((car) => car.id === id);
+
+  return (
+    <div className="pb-4">
+      <div className="flex flex-col items-center justify-center py-16 md:pb-32">
+        <h1 className=" text-4xl font-bold text-center">Reserved Cars.</h1>
+        <p className=" text-xs text-slate-400">Lists of all reservations.</p>
+      </div>
+      <div className="grid grid-cols-responsive px gap-4 justify-items-center">
+        {/* {JSON.stringify(reservation)} */}
```

Comment:
- Please remove this old piece of commented code to improve the repos cleanliness. 👍